app/mailers/user_cleanup_mailer.rb

```diff
@@ -6,6 +6,6 @@ def cleanup_email(will_archive_users, archived_users)
               ' staan op het punt gearchiveerd te worden'
     subject = 'Er zijn gebruikers gearchiveerd!' if @will_archive_users.empty?
 
-    mail to: 'ict@csvalpha.nl;bestuur@csvalpha.nl', subject:
+    mail to: Rails.application.config.x.privacy_email, subject:
```

Comment:
This is not the same thing

Reply:
Is the change intentional?

Reply:
yess it is intentional, archive has to do with privacy so the privacy team should be notified. this is the entirety of bestuur and most members of the ictcie

Reply:
fair enough